synapse/logging/opentracing.py

```diff
@@ -443,10 +443,14 @@ def start_active_span(
     start_time=None,
     ignore_active_span=False,
     finish_on_close=True,
+    *,
```

Comment:
since this isn't a standard parameter for the underlying `opentracing.tracer.start_active_span`, I wanted to make it a kw-only param so that we can safely add more positional params to pass through in future.